### ISSUE TITLE
Conversion from String to int

### ISSUE THREAD

**Issue by hans005** [fesch/Structorizer.Desktop#530]
Would be nice to have a function for this conversion!
Even better of course would be to have conversion from String to floating point as well :-)

Great program already though!!!

**Comment by fesch** [comment]
Integer.valueOf(some_string_variable)
Float.valueOf(some_string_variable)

isn't that enough?

**Comment by hans005** [comment]
OK, sorry, I did not understand the documentation of Structorizer!
I did read that

> Elements from Java and Pascal
> 
> As the executor is Java-based, it should understand most Java commands. Besides this, it has been designed to work with basic Pascal syntax.
> 

but I thought it was only about flow control. Now I have to start to experiment what all from Java really works.

So thanks for your answer. It makes Structorizer even much more usefull for me :-)
And my little suggestion trial can of course be closed.

**Comment by codemanyak** [comment]
@hans005 
> It makes Structorizer even much more usefull for me :-)

Yes but you should be aware that the Java-specific instructions won't be exportable to other languages. And some mechanisms and classes were shadowed in order to implement language-independent mechanisms like arrays and record types.
On the other hand, not even all of the built-in functions and procedures for Structorizer can be translated sensibly to all offered export languages. But we hope to increase the coverage step by step.

**Comment by GitMensch** [comment]
@codemanyak Do you intend to add internal conversion routines? If yes: please adjust the topic, if not: you may want to close this.
